src/views/admin/LeaderBoard.vue

```diff
@@ -246,18 +313,73 @@
 </template>
 <script>
 import Sidebar from "@/components/Sidebar/Sidebar.vue";
+import { get, post } from "../../utilities/api";
 
 export default {
   data() {
-    return {};
+    return {
+      apiURLs: {
+        getDSAs: `/api/get-users?role=18&stats=true`,
+      },
+      agents: [],
+      OId: 0,
+      sortedAgents: [],
+      totalRegistered: 0,
+      option: ''
+    };
   },
   components: {
     Sidebar,
   },
+  async created() {
+    await this.getAgents();
+    this.sortByPurchased();
```

Comment:
set this.option to purchased  and call changeOrder method instead

Reply:
ok